include/boost/geometry/formulas/karney_inverse.hpp

```diff
@@ -465,12 +463,12 @@ class karney_inverse
         {
             if (BOOST_GEOMETRY_CONDITION(CalcFwdAzimuth))
             {
-                result.azimuth = atan2(sin_alpha1, cos_alpha1) * r2d;
+                result.azimuth = atan2(sin_alpha1, cos_alpha1);
```

Comment:
Was this a bug? A leftover from the previous version returning degrees?

Reply:
Probably the second.